community-roadmap/frontend/README.md

```diff
@@ -0,0 +1,32 @@
+# Community Roadmap for Zendesk Help Center
+
+This folder contains the frontend for the Zendesk community roadmap project.
+The frontend is built with npm and React.
+
+## Development
+
+The service can be run locally using `npm`.
+
+Run:
+```
+npm start
+```
+
+## Build
+
+Build a distributable script using
+
+```
+npm run build
+```
+
+## Help Center deployment
+
+Create a custom page and add this content to it:
+
+```
+<div class="container">
+  <div class="roadmapContainer" data-product-key="guide"></div>
+</div>
+<script src="{link to script URL}"></script>
+```
```

Comment:
We seem to be missing a step here. Don't users need to upload the script as an asset first?

In that case, I think we can use an asset helper for the `src` value here.

Reply:
Moved some of this to the main/outer README